### ISSUE TITLE
[Feature Request] Sort Title based on List Order

### ISSUE THREAD

**Issue by zhdenny** [mza921/Plex-Auto-Collections#187]
Would it be possible to implement the ability to maintain order of a Trakt List in a collection?

This would be incredibly useful for Collections like "Top 100 Trending Movies" and such. In a collection like that People want to know what the #1 and #2 and #3 (and so on) most trending title is vs the Alpha/ReleaseDate sort.

**Comment by mza921** [comment]
I don't think this is currently possible, since as you mentioned, Plex only supports sorting collections by release date or alphabetically (but it looks like custom ordering might be [in the works](https://forums.plex.tv/t/collection-order-custom-new/641518).

**Comment by LeendertPlas** [comment]
i wouldn't get your hopes up as the last news about this is from october last year. Plex is vey slow.

**Comment by zhdenny** [comment]
What about using the "Sort Title" feature in Plex somehow? Not sure if that setting is shared between the Collection and the actual Library itself though.
